src/main/java/com/gmail/inverseconduit/commands/sets/AdminCommands.java

```diff
@@ -27,14 +16,7 @@ public static CommandHandle addAdminCommand(BotConfig bc) {
 			String[] args = message.getMessage().trim().split(" ");
 			try {
 				Long newUserId = Long.parseLong(args[1]);
-				if (Permissions.isAdmin(newUserId, bc))
-					Permissions.addAdmin(newUserId, bc);
-				else {
-					return "User already has elevated privileges!";
-				}
-				Path file = Paths.get("bot.properties");
-				addAdminToFile(file, (long) newUserId);
-				return "Added userID " + newUserId + " successful!";
+				return Permissions.addAdmin(newUserId, bc);

```

Comment:
this is a bit of a bad idea, because it intermingles application logic (adding the user to admins) with display logic (notifying the user)